src/main/java/de/doubleslash/keeptime/view/ReportController.java

```diff
@@ -122,6 +125,8 @@ public class ReportController {
 
    private final TreeItem<TableRow> rootItem = new TreeItem<>();
 
+   private static boolean expanded = true;
```

Comment:
can you use this variable also in
projectRow.setExpanded(true); 
instead of hardoced true?